web/public/graph.mjs

```diff
@@ -657,6 +659,10 @@ GraphNode.prototype.has_icon = function() {
   return typeof this.icon !== 'undefined';
 }
 
+GraphNode.prototype.has_comment = function() {
```

Comment:
I'm a bit mixed on adding the "comment triangle" on the pipeline graph.

To me, the pipeline graph is less concerned with why things are failing, and instead is mainly a view to show you what is failing - and if you want to know "why", you can easily drill into the builds page for the latest build.

On the flip-side, it could be useful to see at a glance what failing jobs have already been looked into (by the presence of a comment) without having to drill into the builds page.

But to be honest, I'm leaning toward not adding it to the pipeline page. Thoughts on at least leaving it out of this PR - perhaps this can be implemented further and discussed in a separate PR in the future?